cmd/kube-aggregator/pkg/cmd/server/start.go

```diff
@@ -172,9 +179,8 @@ func (o DiscoveryServerOptions) RunDiscoveryServer() error {
 
 // RunLegacyDiscoveryServer runs the legacy mode of discovery
 func (o DiscoveryServerOptions) RunLegacyDiscoveryServer() error {
-	configFilePath := "config.json"
 	port := "9090"
-	s, err := legacy.NewDiscoverySummarizer(configFilePath)
+	s, err := legacy.NewDiscoverySummarizer(o.ConfigFilePath)
```

Comment:
Oh, we got permission to straight up remove this.  Just delete the entire codepath.